setup.py

```diff
@@ -142,17 +148,29 @@
     ]
     libraries = libraries + ['rt']
     AEROSPIKE_C_TARGET = AEROSPIKE_C_HOME + '/target/Linux-' + machine
+elif WINDOWS:
+    AEROSPIKE_C_TARGET = AEROSPIKE_C_HOME
+    libraries.clear()
+    extra_compile_args.append("-DAS_SHARED_IMPORT")
+    include_dirs.append(AEROSPIKE_C_TARGET + "/vs/packages/aerospike-client-c-dependencies.1.0.2/build/native/include")
```

Comment:
Will this path ever change? For example might the `1.0.2` change with an update? If so maybe this path needs to be built dynamically.

Reply:
Done